cmake/FindDPCPP.cmake

```diff
@@ -45,6 +45,8 @@ if(NOT "${DPCPP_SYCL_TARGET}" STREQUAL "")
   list(APPEND DPCPP_FLAGS "-fsycl-targets=${DPCPP_SYCL_TARGET};")
 endif()
 
+list(APPEND DPCPP_FLAGS "-fno-sycl-instrument-device-code")
```

Comment:
Why is this required? a comment would be good.
Since this seems to be related to linking vtune libraries, maybe there should be an `option` to enable it

Reply:
Yes, that is for linking to VTune. It is a good idea to put an option  here. I will add